src/main/java/frc/robot/commands/drive/SetDrivePerspectiveRobotOriented.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.commands.drive;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Controllers;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drive;
+
+/** Sets the current drive perspective to robot oriented. */
+public class SetDrivePerspectiveRobotOriented extends Command {
+  /** Creates a new SetDrivePerspectiveFieldOriented. */
+  private Drive drive;
+
+  public SetDrivePerspectiveRobotOriented() {
+    this.drive = RobotContainer.drive;
+    addRequirements(drive);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    drive.setDefaultCommand(RobotContainer.driveRobotOriented);
+    Controllers.pilotController.leftTrigger().toggleOnTrue(RobotContainer.driveRobotOriented);
```

Comment:
You want to set it so that driveRobotOrientedSprint runs while left bumper is held. Look at the old code, don't randomly change things for no reason.